testing/separated/ffi-support-extra-test/src/test.rs

```diff
@@ -1,89 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-//! This test is a stress test meant to trigger some bugs seen prior to the use
-//! of handlemaps. See /docs/design/test-faster.md for why it's split -- TLDR:
-//! it uses rayon and is hard to rewrite with normal threads.
```

Comment:
FYI I checked, and this test does indeed seem to have been ported over to the standalone repo, so I don't feel bad about deleting it here.